src/Modals/InteractiveModal/InteractiveModal.ui.js

```diff
@@ -216,3 +217,51 @@ export class InteractiveModal extends Component<Props> {
     )
   }
 }
+
+export type InteractiveModalOpts = {
+  title?: string,
+  message?: string,
+  icon: Component,
+  errorMessage?: string,
+  onPositive: Function,
+  positiveButtonText: string,
+  onNegative: Function,
+  negativeButtonText: string
+}
```

Comment:
`onYes` / `onNo` + `yesButtonText` / `noButtonText`, assuming we call this modal `yesNoModal`